samples/CookieSharing/README.md

```diff
@@ -0,0 +1,19 @@
+# Cookie Sharing Sample App
+
+The sample illustrates cookie sharing across three apps that use cookie authentication:
+
+| Project                                 | Description |
+| --------------------------------------- | ----------- |
+| CookieSharing.Core                      | ASP.NET Core 2.0 Razor Pages app without using ASP.NET Core Identity |
+| CookieSharing.WithIdentity.Core         | ASP.NET Core 2.0 MVC app with ASP.NET Core Identity |
+| CookieSharing.WithIdentity.NETFramework | ASP.NET Framework 4.6.1 MVC app with ASP.NET Identity |
```

Comment:
I understand sharing a login between WithIdentity.Core and WithIdentity.NETFramework, but why do you have to register twice? Wouldn't you also want to share the user database between the two?

Also, sharing a cookie between identity and non-identity apps doesn't make much sense. That Core non-Identity sample should be sharing with a .NET non-Identity sample.

Reply:
I think each of these topics is complicated enough that its reasonable to split them up into Cookie Sharing, and in the future if we really need the shared identity database with old + new mvc identity sample

Reply:
Then why are we providing this sample? Sharing the cookies doesn't seem that useful if the information behind them isn't the same.

Reply:
We've definitely gotten asks about how to share cookies before, and its also a required part of any kind of interop between old/new apps.  This part is the same excluding identity, interop with identity is just a more advanced variation that requires additional stuff to make identity talk to the same db.

Reply:
This sample is focused on sharing the same auth cookie across apps, but not focusing on how you create the cookie/claims.

Reply:
It doesn't make a lot of sense in isolation. Maybe I'm too far removed from the customer requests. Have any backreferences? #34 doesn't have any.

Reply:
I don't remember the issues off the top of my head, but specifically these samples are for this doc: https://docs.microsoft.com/en-us/aspnet/core/security/cookie-sharing?view=aspnetcore-2.1&tabs=aspnetcore2x

Reply:
I plan to remove the samples from the docs repo and perform cross-repo code block sample links. 

How many times have I done this?

![sulu-first-attempt](https://user-images.githubusercontent.com/1622880/39827659-4268ad46-537e-11e8-9bad-8efb143ec8cc.png)

*Actually, it's my first attempt.* - Mr. Sulu ([George Takei](https://georgetakei.com/))
[Star Trek V: The Final Frontier](http://www.startrek.com/), ©1989 [Paramount Pictures](http://www.paramount.com/movies/star-trek-v-final-frontier)